src/main/resources/bidder-config/emx_digital.yaml

```diff
@@ -0,0 +1,20 @@
+adapters:
+  emx_digital:
```

Comment:
Personaly, i'd like to use `emxdigital` as name for bidder in related java code, configs, etc.
But `emx_digital` - only for uid cookie family name.